e2e/tests/localAuthoritySubmitsApplication_test.js

```diff
@@ -0,0 +1,462 @@
+  I.seeEventSubmissionConfirmation(config.applicationActions.enterAttendingHearing);
+  caseViewPage.selectTab(caseViewPage.tabs.legalBasis);
+  I.seeAnswerInTab(1, 'Attending the hearing', 'Interpreter', 'Yes');
+  I.seeAnswerInTab(2, 'Attending the hearing', 'Give details including person, language and dialect', 'French translator');
+  I.seeAnswerInTab(3, 'Attending the hearing', 'Spoken or written Welsh', 'No');
+  I.seeAnswerInTab(4, 'Attending the hearing', 'Intermediary', 'No');
+  I.seeAnswerInTab(5, 'Attending the hearing', 'Facilities or assistance for a disability', 'Yes');
+  I.seeAnswerInTab(6, 'Attending the hearing', 'Give details', 'learning difficulty');
+  I.seeAnswerInTab(7, 'Attending the hearing', 'Separate waiting room or other security measures', 'Yes');
+  I.seeAnswerInTab(8, 'Attending the hearing', 'Give details', 'Separate waiting rooms');
+  I.seeAnswerInTab(9, 'Attending the hearing', 'Something else', 'Yes');
+  I.seeAnswerInTab(10, 'Attending the hearing', 'Give details', 'I need this for this person');
+});
+
+Scenario('local authority uploads documents', (I, caseViewPage, uploadDocumentsEventPage) => {
+  caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
+  uploadDocumentsEventPage.selectSocialWorkChronologyToFollow(config.testFile);
+  uploadDocumentsEventPage.uploadSocialWorkStatement(config.testFile);
+  uploadDocumentsEventPage.uploadSocialWorkAssessment(config.testFile);
+  uploadDocumentsEventPage.uploadCarePlan(config.testFile);
+  uploadDocumentsEventPage.uploadSWET(config.testFile);
+  uploadDocumentsEventPage.uploadThresholdDocument(config.testFile);
+  uploadDocumentsEventPage.uploadChecklistDocument(config.testFile);
+  uploadDocumentsEventPage.uploadAdditionalDocuments(config.testFile);
+  I.continueAndSave();
+  I.seeEventSubmissionConfirmation(config.applicationActions.uploadDocuments);
+  caseViewPage.selectTab(caseViewPage.tabs.documents);
+  I.seeDocument('Social work chronology', '', 'To follow', 'mock reason');
+  I.seeDocument('Social work statement and genogram', 'mockFile.txt', 'Attached');
+  I.seeDocument('Social work assessment', 'mockFile.txt', 'Attached');
+  I.seeDocument('Care plan', 'mockFile.txt', 'Attached');
+  I.seeDocument('Social work evidence template (SWET)', 'mockFile.txt', 'Attached');
+  I.seeDocument('Threshold document', 'mockFile.txt', 'Attached');
+  I.seeDocument('Checklist document', 'mockFile.txt', 'Attached');
+});
+
+Scenario('local authority tries to submit without giving consent', (I, caseViewPage) => {
+  caseViewPage.goToNewActions(config.applicationActions.submitCase);
+  I.see(`I, ${config.swanseaLocalAuthorityUserOne.forename} ${config.swanseaLocalAuthorityUserOne.surname}, believe that the facts stated in this application are true.`);
+  I.click('Continue');
+  I.seeInCurrentUrl('/submitApplication');
+});
+
+Scenario('local authority submits after giving consent', (I, caseViewPage, submitApplicationEventPage) => {
```

Comment:
We rely on the order, are we 100% sure runner is not going to change order in which tests are run? Are we sure the last one is going to be run last?

Reply:
Bit like trying to predict future. I only know it was never the case in the past for a long time. 

Framework is based on mocha which does not support random test ordering.

Reply:
Ok that's fine I was not aware.